modules/veteran/spec/requests/v0/other_accredited_representatives_spec.rb

```diff
@@ -0,0 +1,117 @@
+
+      create(:representative, representative_id: '234', poa_codes: ['A12'], user_types: ['attorney'],
+                              long: -77.436649, lat: 39.101481, location: 'POINT(-77.436649 39.101481)',
+                              first_name: 'Eliseo', last_name: 'Schroeder') # ~25 miles from Washington, D.C.
+
+      create(:representative, representative_id: '345', poa_codes: ['A12'], user_types: ['attorney'],
+                              long: -76.609383, lat: 39.299236, location: 'POINT(-76.609383 39.299236)',
+                              first_name: 'Marci', last_name: 'Weissnat') # ~35 miles from Washington, D.C.
+
+      create(:representative, representative_id: '456', poa_codes: ['A12'], user_types: ['attorney'],
+                              long: -77.466316, lat: 38.309875, location: 'POINT(-77.466316 38.309875)',
+                              first_name: 'Gerard', last_name: 'Ortiz') # ~47 miles from Washington, D.C.
+
+      create(:representative, representative_id: '567', poa_codes: ['A12'], user_types: ['attorney'],
+                              long: -76.3483, lat: 39.5359, location: 'POINT(-76.3483 39.5359)',
+                              first_name: 'Adriane', last_name: 'Crona') # ~57 miles from Washington, D.C.
+    end
+
+    it 'does not include representatives outside of max distance' do
+      get path, params: { type:, lat: 38.9072, long: -77.0369 }
+
+      parsed_response = JSON.parse(response.body)
+
+      expect(parsed_response['data'].pluck('id')).not_to include('567')
+    end
+
+    it 'sorts by distance_asc by default' do
+      get path, params: { type:, lat: 38.9072, long: -77.0369 }
+
+      parsed_response = JSON.parse(response.body)
+
+      expect(parsed_response['data'].pluck('id')).to eq(%w[123 234 345 456])
+    end
+
+    it 'can sort by first_name_asc' do
+      get path, params: { type:, lat: 38.9072, long: -77.0369, sort: 'first_name_asc' }
+
+      parsed_response = JSON.parse(response.body)
+
+      expect(parsed_response['data'].pluck('id')).to eq(%w[123 234 456 345])
+    end
+
+    it 'can sort by first_name_desc' do
+      get path, params: { type:, lat: 38.9072, long: -77.0369, sort: 'first_name_desc' }
+
+      parsed_response = JSON.parse(response.body)
+
+      expect(parsed_response['data'].pluck('id')).to eq(%w[345 456 234 123])
+    end
+
+    it 'can sort by last_name_asc' do
+      get path, params: { type:, lat: 38.9072, long: -77.0369, sort: 'last_name_asc' }
+
+      parsed_response = JSON.parse(response.body)
+
+      expect(parsed_response['data'].pluck('id')).to eq(%w[123 456 234 345])
+    end
+
+    it 'can sort by last_name_desc' do
+      get path, params: { type:, lat: 38.9072, long: -77.0369, sort: 'last_name_desc' }
+
+      parsed_response = JSON.parse(response.body)
+
+      expect(parsed_response['data'].pluck('id')).to eq(%w[345 234 456 123])
+    end
+
+    it 'can sort fuzzy search on name' do
```

Comment:
This is just a typo probably from me copy + pasting descriptions in the original spec. Not a blocker at all.

```suggestion
    it 'can fuzzy search on name' do
```

Reply:
TY

Reply:
Fixed